src/main/scala/gcd/GCD.scala

```diff
@@ -32,3 +33,14 @@ class GCD extends Module {
   io.outputGCD := x
   io.outputValid := y === 0.U
 }
+
+/**
+ * Generate Verilog sources and save it in file GCD.v
+ */
+object GCD extends App {
+  val verilog_src = ChiselStage.emitSystemVerilog(
```

Comment:
Would it not be simpler to call ChiselStage.emitSystemVerilogFile rather than emitting as a string and then writing?

Reply:
Yes of course ! I didn't know about this function.
That's why it's useful to provide an example :)